### PR TITLE
Added documentation for virtual event (P0)

### PR DESCRIPTION
**Pull request by frankpeng7**
Added documentation for virtual event (P0 cases):
- all related entities, enums and complex types
-  two new permissions
- API: Get virtualEventWebinar
- API: Get virtualEventSession
- API: Get virtualEventRegistrant

### CODE REVIEW

File: api-reference/beta/api/virtualeventregistrant-get.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+      {
+        "questionId": "4577afdb-8bee-4219-b482-04b52c6b855c",
+        "displayName": null,
+        "value": null,
+        "booleanValue": true,
+        "multiChoiceValues": []
+      },
+      {
+        "questionId": "80fefcf1-caf7-4cd3-b8d7-159e17c47f20",
+        "displayName": null,
```
Comment (satyakonmsft) [inline api-reference/beta/api/virtualeventregistrant-get.md:114]: can we give samples of questions? assuming displayName is the actual question?

**Comment by dodaromike** [comment]
@frankpeng7 These new documents will need changelog entries.  Please see https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog

**Comment by dodaromike** [comment]
@frankpeng7  Thanks for CSDL and API.md.  I noticed that both changelog and TOC entries are needed for these changes.

**Comment by frankpeng7** [comment]
@dodaromike changelog is included in the PR. Let me add TOC in the next revision

**Comment by dodaromike** [comment]
Changelog and TOC entries included.

**Review by dodaromike — APPROVED** [review]

File: api-reference/beta/api/attendancerecord-list.md [hunk 2]
```diff
@@ -259,3 +273,74 @@ Content-Type: application/json
   ]
 }
 ```
+
+### Example 3: List attendance records of a virtual event attendance report
```
Comment (frankpeng7) [inline api-reference/beta/api/attendancerecord-list.md:277]: VE session

File: api-reference/beta/api/attendancerecord-list.md [hunk 3]
```diff
@@ -38,6 +46,12 @@ GET /me/onlineMeetings/{meetingId}/attendanceReports/{reportId}/attendanceRecord
 GET /users/{userId}/onlineMeetings/{meetingId}/attendanceReports/{reportId}/attendanceRecords
 ```
 
+To get attendance records of a virtual event's attendance report:
```
Comment (frankpeng7) [inline api-reference/beta/api/attendancerecord-list.md:49]: Might be more clear to say ve session's attendance report

File: api-reference/beta/api/attendancerecord-list.md (hunk 2 above)
Comment (yuyaolian-msft) [reply api-reference/beta/api/attendancerecord-list.md]: file updated per comment.

File: api-reference/beta/api/attendancerecord-list.md (hunk 3 above)
Comment (yuyaolian-msft) [reply api-reference/beta/api/attendancerecord-list.md]: file updated per comment.

File: api-reference/beta/resources/virtualeventwebinar.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+| audience | [meetingAudience](#meetingaudience-values) | To whom the webinar is visible. |
+| coOrganizers  | [communicationsUserIdentity](communicationsuseridentity.md) collection | Identity information of co-organizers of the webinar. |
+| createdBy | [communicationsIdentitySet](communicationsidentityset.md) | Identity information of who created the webinar. Inherited from [virtualEvent](../resources/virtualevent.md). |
+| description | String | Description of the webinar. Inherited from [virtualEvent](../resources/virtualevent.md). |
+| displayName | String | Display name of the webinar. Inherited from [virtualEvent](../resources/virtualevent.md). |
+| endDateTime | [dateTimeTimeZone](../resources/datetimetimezone.md) | End time of the webinar. Inherited from [virtualEvent](../resources/virtualevent.md). |
+| startDateTime | [dateTimeTimeZone](../resources/datetimetimezone.md) | Start time of the webinar. Inherited from [virtualEvent](../resources/virtualevent.md). |
+| id | String | Unqiue identifier of the webinar. Inherited from [entity](../resources/entity.md).|
+| status | [virtualEventStatus](#virtualeventstatus-values) | Status of the webinar. |
```
Comment (Danielabom) [inline api-reference/beta/resources/virtualeventwebinar.md:38]: This property is not inherited and was not in the CSDL. Please check.

File: api-reference/beta/resources/virtualeventsession.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+doc_type: resourcePageType
+---
+
+# virtualEventSession resource type
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Represents a session of a [virtualEvent](../resources/virtualevent.md)
```
Comment (Danielabom) [inline api-reference/beta/resources/virtualeventsession.md:16]:
```suggestion
Represents a session in a [virtualEvent](../resources/virtualevent.md)
```

File: api-reference/beta/resources/virtualeventwebinar.md (hunk 4 above)
Comment (frankpeng7) [reply api-reference/beta/resources/virtualeventwebinar.md:38]: It is inherited from entity `virtualEvent` which has this property `status`

**Review by Danielabom — APPROVED** [review]
Thank you for working on this PR @frankpeng7 and @dodaromike. PR approved.

**Comment by frankpeng7** [comment]
Hi @Danielabom sorry for the late notice but we had to add 4 more docs to this batch. Can you please review them? Changes after your approval are listed here: https://github.com/microsoftgraph/microsoft-graph-docs/pull/20889/commits/00923ea20de29fa0234f2bacfcec1ba67354b492

**Review by satyakonmsft — APPROVED** [review]

**Pull request merged**